device/accton/x86_64-accton_csp7551-r0/Accton-CSP7551/switch-tna-sai.conf

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
The content of this file has been updated recently. To make it compatible with the latest SDK, I'd recommend to use the latest. E.g., https://github.com/Azure/sonic-buildimage/blob/761ae244271d01f60bb6a4462888590066f78338/device/barefoot/x86_64-accton_as9516_32d-r0/newport/switch-tna-sai.conf#L1

Reply:
Ok, I'll update it.